modules/ml/src/main/java/org/apache/ignite/ml/math/impls/matrix/DenseLocalOnHeapMatrix.java

```diff
@@ -128,6 +128,6 @@ private DenseLocalOnHeapMatrix(DenseLocalOnHeapMatrix orig, int acsMode) {
 
     /** */
```

Comment:
{@inheritDoc} seems to be missing here, compare to javadocs in other methods of this class